cluster/gce/util.sh

```diff
@@ -743,8 +745,9 @@ function construct-common-kubelet-flags {
 # Sets KUBELET_ARGS with the kubelet flags for Linux nodes.
 # $1: if 'true', we're rendering flags for a master, else a node
 function construct-linux-kubelet-flags {
-  local node_type="$1"
-  local flags="$(construct-common-kubelet-flags)"
+  local node_type=$1
+  local flags
+  flags=$(construct-common-kubelet-flags)
```

Comment:
why drop the quotes?

Reply:
Quotes are superfluous on assignments.

Reply:
Re-added quotes according to previous discussion (in other PR).